### PR TITLE
add case study for zcitc

### PR DESCRIPTION
**Pull request by wbc6080**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs update

* **What is the new behavior (if this is a feature change)?**
Add Case for zcitc in smart parking

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-pages/case-studies/zcitc/index.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+date: 2023-12-25
+title: 浙江志诚软件有限公司
+subTitle: 云原生边缘计算KubeEdge在智慧停车中的实践
+description: 在智慧停车事业蓬勃发展的今天，开放式一体化智慧停车操作系统承载的停车场等边缘计算节点数量快速增长，给管理运维带来巨大挑战。为响应国家信创战略，适应云原生开发新时代，志诚团队应用一种基于 Kubernetes 架构的云原生边缘计算系统 KubeEdge，很好地解决了停车场场景下应用编排、部署、更新等难题，为城市停车资源整合提供了生动案例。
```
Comment (fisherxu) [inline i18n/zh/docusaurus-plugin-content-pages/case-studies/zcitc/index.mdx:5]: 太长了，精简一下

File: src/pages/case-studies/zcitc/index.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+date: 2023-12-02
+title: zcitc
```
Comment (fisherxu) [inline src/pages/case-studies/zcitc/index.mdx:3]:
```suggestion
title: ZCITC
```

File: src/pages/case-studies/zcitc/index.mdx [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+subTitle: The practice of KubeEdge in smart parking
+description: The zcitc team applied KubeEdge, a cloud-native edge computing system based on the Kubernetes architecture, to effectively solve the problems of application orchestration, deployment, and update in parking lot scenarios.
+tags:
+  - UserCase
+---
+
+# zcitc practice in smart parking base on KubeEdge
```
Comment (fisherxu) [inline src/pages/case-studies/zcitc/index.mdx:10]:
```suggestion
# ZCITC practice in smart parking base on KubeEdge
```

File: src/pages/case-studies/zcitc/index.mdx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+date: 2023-12-02
+title: zcitc
+subTitle: The practice of KubeEdge in smart parking
+description: The zcitc team applied KubeEdge, a cloud-native edge computing system based on the Kubernetes architecture, to effectively solve the problems of application orchestration, deployment, and update in parking lot scenarios.
```
Comment (fisherxu) [inline src/pages/case-studies/zcitc/index.mdx:5]:
```suggestion
description: The ZCITC team applied KubeEdge to effectively solve the problems of application orchestration, deployment, and update in parking lot scenarios.
```

File: i18n/zh/docusaurus-plugin-content-pages/case-studies/zcitc/index.mdx [hunk 5]
```diff
@@ -0,0 +1,85 @@
+---
+date: 2023-12-25
+title: 浙江志诚软件有限公司
+subTitle: 云原生边缘计算KubeEdge在智慧停车中的实践
```
Comment (Shelley-BaoYue) [inline i18n/zh/docusaurus-plugin-content-pages/case-studies/zcitc/index.mdx:4]: subtitle适合写项目名，如果没有的话，可以不写

File: src/pages/case-studies/zcitc/index.mdx (hunk 4 above)
Comment (wbc6080) [reply src/pages/case-studies/zcitc/index.mdx]: ALL done

File: i18n/zh/docusaurus-plugin-content-pages/case-studies/zcitc/index.mdx (hunk 5 above)
Comment (wbc6080) [reply i18n/zh/docusaurus-plugin-content-pages/case-studies/zcitc/index.mdx]: Done

File: i18n/zh/docusaurus-plugin-content-pages/case-studies/zcitc/index.mdx [hunk 6]
```diff
@@ -0,0 +1,85 @@
+---
+date: 2023-12-25
+title: 浙江志诚软件有限公司
+subTitle:
+description: 志诚团队应用KubeEdge，解决了停车场场景下应用编排、部署、更新等难题，为城市停车资源整合提供了案例。
```
Comment (fisherxu) [inline i18n/zh/docusaurus-plugin-content-pages/case-studies/zcitc/index.mdx:5]:
```suggestion
description: 志诚团队使用KubeEdge，解决了停车场场景下应用编排、部署、更新等难题，提升了城市停车资源整合管理效率。
```

Comment (wbc6080) [reply i18n/zh/docusaurus-plugin-content-pages/case-studies/zcitc/index.mdx]: DONE

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**